### PR TITLE
La

### PR DESCRIPTION
**Pull request by InnocentBug**
# Description

## Checklist

- [ ] My name is on the list of contributors (`CONTRIBUTORS.md`) in the pull request source branch.
- [ ] I have updated the documentation to reflect my changes.

### CODE REVIEW

File: src/cript/api/__init__.py [hunk 1]
```diff
@@ -3,3 +3,6 @@
 from cript.api.api import API
 from cript.api.valid_search_modes import SearchModes
 from cript.api.vocabulary_categories import VocabCategories
+
+# Default maximum seconds for all requests to the backend
+_API_TIMEOUT = 25
```
Comment (nh916) [inline src/cript/api/__init__.py:8]: how come we are adding this?

Comment (InnocentBug) [reply src/cript/api/__init__.py:8]: The new linter tool `bandit` is complaining if you don't have a time out.
And I think they have a good point, so adding the new version of trunk adds this.

**Comment by InnocentBug** [comment]
> is this a real PR that we want merged or just to illustrate a point?

You asked to look into fixing an issue.
And this PR does that.
Merge it if you want, you don't have to.

There is a real, real PR to bump up trunks version #299

File: src/cript/api/__init__.py (hunk 1 above)
Comment (nh916) [reply src/cript/api/__init__.py:8]: I think the API and HTTP times out at some point anyways and I think some of the search like bigsmiles search takes a while for the response to come back which might create issues if our timeout kicks in sooner than it has time to respond.

Did we have bandit before?

Comment (InnocentBug) [reply src/cript/api/__init__.py:8]: No Bandit is new.
We don't have a timeout set before. And that means, that connections might hang forever.

It is good practice to set one:
https://requests.readthedocs.io/en/latest/user/advanced/#timeouts

File: src/cript/api/exceptions.py [hunk 2]
```diff
@@ -129,6 +130,35 @@ def __str__(self) -> str:
         return error_message
 
 
+class CRIPTDuplicateNameError(CRIPTAPISaveError):
```
Comment (nh916) [inline src/cript/api/exceptions.py:133]: is this block supposed to be in this PR?

Comment (InnocentBug) [reply src/cript/api/exceptions.py:133]: It shows up, because branch is not up to date with `develop`.
But I based the trunk update on the new `develop`.

**Review by nh916 — APPROVED** [review]

**Comment by nh916** [comment]
I fear that merging this into both `delete` and `develop` branch may end up with merge conflicts.

I wonder if it'll be better if we merge this to `develop` and then I do a `git merge` to get the latest changes into the `delete` branch?

**Comment by nh916** [comment]
I'm seeing a circular import error from our tests, I would recommend maybe creating a `api_configs.py` file and putting all configuration variables in there to avoid the import error and have it be nice and neat. We can add some comments and maybe docstring at the top too to make it even cleaner and nicer!

> Love how our tests can catch errors for us!

**Pull request closed without merging**